### PR TITLE
add email notification in `email-composer` and `email-sender` services

### PR DESCRIPTION
**Pull request by JoaoVictor6**

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "NEXT_PUBLIC_POSTHOG_KEY",
     "SUPABASE_URL",
     "SUPABASE_SERVICE_ROLE",
-    "NEXT_PUBLIC_POSTHOG_HOST"
+    "NEXT_PUBLIC_POSTHOG_HOST",
+    "RESEND_KEY"
```
Comment (ocodista) [inline turbo.json:37]: Should we add this to the .env.example too?

How can we add the env so AWS EC2 also have it?

**Review by ocodista — APPROVED** [review]
Please address the comments before merging it @JoaoVictor6

File: turbo.json (hunk 1 above)
Comment (JoaoVictor6) [reply turbo.json:37]: > How can we add the env so AWS EC2 also have it?
In this moment we access EC2 and config envs manually

File: apps/web/.env.example [hunk 2]
```diff
@@ -14,4 +14,8 @@ OWNER_EMAIL=
 NEXT_PUBLIC_MIXPANEL_KEY=
 
 # Random string
-CRON_SECRET=
\ No newline at end of file
+CRON_SECRET=
+
+# POSTHOG
+NEXT_PUBLIC_POSTHOG_HOST=
+NEXT_PUBLIC_POSTHOG_KEY=
```
Comment (ocodista) [inline apps/web/.env.example:21]:
```suggestion

```

**Pull request closed without merging**